conda/third_party/dali_opencv/recipe/meta.yaml

```diff
@@ -19,9 +19,9 @@
     version: {{ build_version }}
 
   source:
-    fn: opencv-4.9.0.tar.gz
-    url: https://github.com/opencv/opencv/archive/refs/tags/4.9.0.tar.gz
-    sha256: ddf76f9dffd322c7c3cb1f721d0887f62d747b82059342213138dc190f28bc6c
+    fn: opencv-4.10.0.tar.gz
```

Comment:
Cleary, judging by the version in the libjpeg-turbo, it does not currently matter, but I think those versions were meant to be reflected in `set build_version` (L15I in both files.